app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -32,14 +36,29 @@ public Product getProduct(Long id) {
         return findProduct(id);
     }
 
-    public Product createProduct(ProductData productData) {
+    public Product createProduct(ProductData productData, Authentication authentication) {
         Product product = mapper.map(productData, Product.class);
+        product.setCreator(mapper.map(authentication, User.class));
+
         return productRepository.save(product);
     }
 
-    public Product updateProduct(Long id, ProductData productData) {
+    /**
+     * 상품을 수정하고 수정된 정보를 리턴한다.
+     *
+     * @param id 수정할 상품의 식별자
+     * @param productData 수정할 상품의 목표 정보
+     * @param authentication 권한 정보
+     * @return 수정된 상품 정보
+     * @Throw 권한이 없을 경우
```

Comment:
어떤 예외 타입을 던지는지 적어주도록 합시다.
```suggestion
     * @Throw InvalidRoleException 권한이 없을 경우
```